src/charts/scale.js

```diff
@@ -151,8 +150,11 @@ export default function Scale () {
 
     let yDomain = null
     if (config.yDomain === "auto") {
+      const allStackHeights = data.dataByKey.map((d) => d3.sum(d.series.map((dB) => dB.value)))
       const valuesExtent = d3.extent(allStackHeights)
       yDomain = [0, valuesExtent[1]]
+    } else if (config.yDomain === "percentage") {
```

Comment:
maybe this should be a utility function or constant so that it could be used elsewhere? For example the `domain-editor` currently assumes `config.yDomain` to be an array, not a string.

Reply:
I think you'll also need to modify `hover.js` to fix the issue with the circles that are displayed on mouse hover.